test/bigtest/network/config.js

```diff
@@ -50,7 +50,18 @@ export default function config() {
   this.get('/users');
   this.get('/material-types');
   this.get('/identifier-types');
-  this.get('/contributor-name-types', () => ({ contributorNameTypes: [{ id: 'test', name: 'Test' }] }));
+  this.get('/contributor-name-types', () => ({
```

Comment:
please make a model, factory and serializer for this, just like identifier-types